src/css/relay.css

```diff
@@ -195,21 +175,34 @@ button.fx-relay-modal-close-button {
   transform: rotate(45deg);
 }
 
-.fx-relay-alias-loading::after {
+.fx-relay-alias-loading {
+  position: relative;
+}
+
+.fx-relay-alias-loading .fx-relay-alias-loading-image {
+  opacity: 1;
+  z-index: 3;
+  animation: fxRelayFadeIn 1s ease;
+  display: block;
+}
+
+.fx-relay-alias-loading-image img {
```

Comment:
This entire CSS block was part of a refactor for the "loading/processing" animation when a user clicks the "Generate alias" from the content script. Previously, it was set via CSS in an `after::`. I had to set the path to the image dynamically via `browser.runtime.getURL` and you cannot set `after::` CSS via JavaScript. This required a refactor to move the animation class to an image, rather.